app/libs/pending_comment.py

```diff
@@ -42,7 +40,7 @@ def create(
     status = comment_statuses.pending.value
     if should_approve():
```

Comment:
- Please rename `should_approve` to `is_autoapproval_on` 
- also it can be simplified to
```
def is_autoapproval_on():
    return settings.MODERATION_POLICY == moderation_policies.automatic.value
```